app/src/main/java/org/oppia/android/app/application/AbstractOppiaApplication.kt

```diff
@@ -51,6 +51,9 @@ abstract class AbstractOppiaApplication(
     // TODO(#4751): Re-enable WorkManager for S+.
     if (Build.VERSION.SDK_INT < Build.VERSION_CODES.S) {
       FirebaseApp.initializeApp(applicationContext)
+      // FirebaseAppCheck protects our API resources from abuse. It works with Firebase services,
+      // Google Cloud services, and can also be implemented for our own APIs.
+      // See https://firebase.google.com/docs/app-check for currently supported Firebase products.
```

Comment:
```suggestion
      // See https://firebase.google.com/docs/app-check for currently supported Firebase products. Note that as of this code being checked in, only the app's Firestore usage is affected by App Check (Analytics is NOT affected).
```

(Needs wrapping, but this adds the additional context that I think we need to include per my earlier comment).